Makefile

```diff
@@ -6,13 +6,12 @@ GO_TEST_EXTRA ?=
 GO_TEST_COVER_PROFILE ?= cover.out
 GO_TEST_CODECOV ?=
 
-TOP_DIR ?= $(shell git rev-parse --show-toplevel)
```

Comment:
Are you sure it would not break for example `make release` command?

Reply:
`TOP_DIR` is not used anywhere

Reply:
> `TOP_DIR` is not used anywhere

and what about other variables?

Reply:
I've changed only those, which required for `GO_BUILD_LDFLAGS`